### PR TITLE
Kotlin: cleanup after internal changes

### PR DESCRIPTION
**Pull request by redsun82**

### CODE REVIEW

**Comment by redsun82** [comment]
ah, forgot it's still used by an internal PR, we need to clean that up before merging this.

**Comment by criemen** [comment]
Hm, CI is still broken. I'm not convinced 😬

**Comment by redsun82** [comment]
> Hm, CI is still broken. I'm not convinced 😬

uh, I wonder whether the internal PR didn't break those. Language tests might not have been triggered on that.

**Comment by redsun82** [comment]
I can't understand how this PR could cause that failure, as if this PR had an influence, it would just break the build, not the language test running. `semmle-code` main is indeed clean, but I've started a run with `semmle-code` main and `codeql` main (simulating a ql bump).

**Comment by redsun82** [comment]
This must wait for https://github.com/github/codeql/pull/16731 and an internal PR

File: java/ql/lib/change-notes/2024-06-13-kotlin-qltest-support.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+category: fix
+---
+* Support of `codeql test run` for Kotlin sources has been fixed.
```
Comment (redsun82) [inline java/ql/lib/change-notes/2024-06-13-kotlin-qltest-support.md:4]: @igfoo does this sound right? Or would you prefer another wording?

**Review by igfoo — DISMISSED** [review]

File: java/ql/lib/change-notes/2024-06-13-kotlin-qltest-support.md (hunk 1 above)
Comment (igfoo) [reply java/ql/lib/change-notes/2024-06-13-kotlin-qltest-support.md:4]: I'd probably have written "Support for".

**Review by criemen — APPROVED** [review]
Re-approving for Ian.

**Pull request merged**